packages/core/lib/services/substrate/service.js

```diff
@@ -46,7 +46,8 @@ export class SubstrateService {
   async getFeeAmount(params: TransactionParams) {
     validation.getFeeAmount(params);
 
-    const {toAddress, fromAddress, amount} = params;
+    const {toAddress, fromAddress} = params;
+    const amount = getPlainDockAmount(params.amount).toNumber();
```

Comment:
any chance `getPlainDockAmount()` could return something that can't be converted to a number? And if so, what's the result? Does it throw an exception? Or return 0 or -1 or something?

Reply:
the `validation.getFeeAmount(params);` will throw in case the amount is not a number

Reply:
There is a check for `isNumberValid` on it

Reply:
are we handling that exception gracefully in the UI already?

Reply:
Thanks, i've added this to dock-app repo https://github.com/docknetwork/dock-app/pull/372